src/Configuration/Option.php

```diff
@@ -194,7 +194,7 @@ final class Option
      * @internal Use @see \Rector\Config\RectorConfig::parallel() instead with pass int $seconds parameter
      * @var string
      */
-    public const PARALLEL_TIMEOUT_IN_SECONDS = 'parallel-timeout-in-seconds';
+    public const PARALLEL_JOB_TIMEOUT_IN_SECONDS = 'parallel-job-timeout-in-seconds';
```

Comment:
@TomasVotruba @samsonasik is this considered a BC break?

Reply:
probably add comment is better than rename constant

Reply:
I realize it's marked as internal, so shouldn't be an issue I think

Reply:
This is only internal constants, so the change should be ok :+1: